### DISCUSSION TITLE
404 NOT_FOUND

### DISCUSSION THREAD

**Discussion by web-dev-pr0** [vercel/community#5731] [Help]
### Summary

I'm trying to deploy my app and when i go to the url i get 404 ive tried some stuff that ive seen but still nothing works

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/132590388/5f908225-649e-4fe4-9f2b-452b7eef899d)

**Comment by palacios-winkler** [comment]
aquí no hay ayuda

**Comment by amyegan** [comment]
Hi @web-dev-pr0. There are many possible causes of a 404, so this isn't enough information to tell us what's really going wrong.

Please share more details about your project, or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), to give us more clues about what might be causing the error. That way someone here may be able to recognize the cause and share potential solutions. 🙏
